application/app/src/main/resources/application.yaml

```diff
@@ -0,0 +1,5 @@
+server:
```

Comment:
Can define the port number in the Makefile profile and pass it as an argument? I also wonder if we should include SSL cert and expose port 8443?

Reply:
Yes, will do.